README.rst

```diff
@@ -165,7 +165,7 @@ Issues
 ------
 
 Please use `Github's issue
-tracker <https://github.com/dssg/johnson-county-ddj-public/issues>`__
+tracker <https://github.com/dssg/johnson-county-ddj-public/issues>`
```

Comment:
Argh! Sorry, the automerge process chopped up that line and messed with the formatting.